starport/interface/cli/starport/cmd/network_chain_show.go

```diff
@@ -28,25 +39,61 @@ func networkChainShowHandler(cmd *cobra.Command, args []string) error {
 
 	chainID := args[0]
 
-	c, err := nb.ShowChain(context.Background(), chainID)
+	// Get flags
+	home, _, err := getHomeFlags(cmd)
 	if err != nil {
 		return err
 	}
-	chainyaml, err := yaml.Marshal(c)
+	showGenesis, err := cmd.Flags().GetBool(genesisFlag)
 	if err != nil {
 		return err
 	}
-
-	info, err := nb.LaunchInformation(context.Background(), chainID)
+	showPeers, err := cmd.Flags().GetBool(peersFlag)
 	if err != nil {
 		return err
 	}
-	infoyaml, err := yaml.Marshal(info)
+	if showGenesis && showPeers {
+		return fmt.Errorf("%s and %s flags cannot be used together", genesisFlag, peersFlag)
+	}
+
+	// Fetch launch information
+	info, err := nb.LaunchInformation(cmd.Context(), chainID)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("\nChain:\n---\n%s\n\nLaunch Information:\n---\n%s", string(chainyaml), string(infoyaml))
+	switch {
+	case showGenesis:
+		// Generate the genesis in a temporary directory and show the content
+		tmpHome, err := nb.GenerateTemporaryGenesis(cmd.Context(), chainID, home, info)
+		defer os.RemoveAll(string(tmpHome))
+		if err != nil {
+			return err
+		}
+		genesis, err := ioutil.ReadFile(tmpHome.GenesisPath())
+		if err != nil {
+			return err
+		}
+		fmt.Print(string(genesis))
+	case showPeers:
+		// Show the peers in the config.toml format
+		fmt.Printf(`persistent_peers = "%s"`, strings.Join(info.Peers, ","))
```

Comment:
```suggestion
		fmt.Printf(`persistent_peers = "%s"\n`, strings.Join(info.Peers, ","))
```

Reply:
I don't know if we should use carriage return in order to keep the output as minimalistic as possible. What do you think @fadeev ?

Reply:
Peers will be copy pasted into config.toml. if we do newlines, users will have to manually insert commas, so I suggest we stick with commas.

Reply:
We have commas, the question is if we should have a newline at the end of the line

Reply:
Yes, sure, we should have a carriage return.